Dockerfile

```diff
@@ -18,4 +41,4 @@ USER appuser
 
 EXPOSE 8000
 
-CMD ["uvicorn", "main:app", "--reload", "--host", "0.0.0.0"]
+CMD ["python", "-m", "uvicorn", "main:app", "--host", "0.0.0.0", "--port", "8000"]
```

Comment:
When try to execute uvicorn CLI in runtime stage it will report that it cannot be found (by default, it looks in /usr/bin/ or /bin), that is because uvicorn was installed during the build stage. However, since all the Python libraries installed, including uvicorn, were copied to the runtime stage from build stage, the command can be found using python -m uvicorn, which points to the tools under /usr/local/lib/python3.12/site-packages/

Reply:
Thank you very much for that clarification! 
And thank you so much for the contribution